rebasehelper/utils.py

```diff
@@ -101,7 +101,15 @@ class ConsoleHelper(object):
 
     @classmethod
     def should_use_colors(cls, conf):
-        """Determine whether ansi colors should be used for CLI output"""
+        """Determine whether ansi colors should be used for CLI output
+
+        Args:
+            conf: configuration object with arguments from the command line
+
+        Return:
+            True or False, depends on whether colors should be used
```

Comment:
Again, missing type (`bool`), and I would write it simply as: `bool: Whether colors should be used.`.

Reply:
Take a look on docstrings in [`KojiHelper`](https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/utils.py#L912), and official [google doc](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments).